react/index.js

```diff
@@ -48,28 +48,26 @@ class Header extends Component {
       linkUrl,
       logoUrl,
       logoTitle,
-      leanMode,
       showSearchBar,
       showLogin,
     }
 
     return (
-      <Fragment>
-        <div
-          className={`vtex-header force-full-width relative z-2 ${leanMode ? 'vtex-header-lean-mode' : ''}`}
-        >
-          <TopMenu
-            {...topMenuOptions}
-            extraHeaders={(
-              <ExtensionPoint id="telemarketing" />
-              // TODO: either add support or remove menu-link
-              // <div className="z-2 items-center w-100 top-0 bg-base tl">
-              //   <ExtensionPoint id="menu-link" />
-              // </div>
-            )}
-          />
-        </div>
-      </Fragment>
+      <div
+        className={`vtex-header force-full-width relative z-2 ${leanMode ? 'vtex-header-lean-mode' : ''}`}
+      >
+        <TopMenu
+          {...topMenuOptions}
+          leanMode={leanMode}
+          extraHeaders={(
+            <ExtensionPoint id="telemarketing" />
+            // TODO: either add support or remove menu-link
+            // <div className="z-2 items-center w-100 top-0 bg-base tl">
```

Comment:
Preserve that comments are really necessary? 🤔

Reply:
not really, I'm afraid it's going to be completely forgotten if it's removed though. Not sure how important it is, however.